### PR TITLE
Get WCT working with Appveyor

### PR DESCRIPTION
**Pull request by usergenic**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Test against this version of Node.js
+environment:
+  matrix:
+    # - nodejs_version: '8'
```
Comment (justinfagnani) [inline appveyor.yml]: Is this comment temporary? What's the context?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+matrix:
+  fast_finish: true
+shallow_clone: true
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - node --version
+  - npm --version
+  # try to limit tests to ie
```
Comment (justinfagnani) [inline appveyor.yml]: "try"?

File: test/integration/browser.ts [hunk 3]
```diff
@@ -430,6 +431,10 @@ describe('early failures', () => {
            },
          },
        };
+       if (process.env.LOCAL_BROWSERS) {
```
Comment (justinfagnani) [inline test/integration/browser.ts]: Is this a new env variable? Does it need to be documented?

**Comment by usergenic** [comment]
@justinfagnani this PR wasn't quite ready for review; I've been using it as a vehicle to try and coerce appveyor to succeeding in running tests on Windows as its behavior is different than the Windows VM I was developing on.  I'll send a review request once I can get CI green for IE11.

**Pull request closed without merging**